wrappers/javascript/nodejs/src/NodeJSIndyCredx.ts

```diff
@@ -60,11 +60,11 @@ export class NodeJSIndyCredx implements IndyCredx {
     return new ObjectHandle(ret.deref() as number)
   }
 
-  public schemaGetAttribute(options: { schema: ObjectHandle; name: string }) {
-    const { schema, name } = serializeArguments(options)
+  public schemaGetAttribute(options: { object: ObjectHandle; name: string }) {
+    const { object, name } = serializeArguments(options)
```

Comment:
I think we should use the word `handle` or `objectHandle` here. `object` is also an internal keyword and might cause some annoying issues